cmd/ocm-backplane/cloud/common.go

```diff
@@ -221,6 +221,15 @@ type namedRoleArn struct {
 }
 
 func (cfg *QueryConfig) getIsolatedCredentials(ocmToken string) (aws.Credentials, error) {
+	const (
+		productionOCMUrl            = "https://api.openshift.com"
+		productionAssumeInitialArn  = "arn:aws:iam::922711891673:role/SRE-Support-Role"
+		stagingOCMUrl               = "https://api.stage.openshift.com"
+		stagingAssumeInitialArn     = "arn:aws:iam::277304166082:role/SRE-Support-Role"
+		integrationOCMUrl           = "https://api.integration.openshift.com"
+		integrationAssumeInitialArn = "arn:aws:iam::277304166082:role/SRE-Support-Role"
```

Comment:
I wouldn't recommend exposing internal API URLs in a public repository, that's the reason why we had to fallback to leverage CS environment resource for automation instead of doing it via the CLI, more details in https://issues.redhat.com/browse/OCM-1534.

Reply:
Right, those are the backplane API URLs though, which I purposely avoided here

Reply:
Thanks, I see it is safe to expose the ARNs publicly as well going by the Jira.

My initial assumption was they're something sensitive.

Reply:
No worries, good to make sure since it looks very similar!